cli/cli/Services/BeamoLocalSystem.cs

```diff
@@ -136,6 +136,52 @@ public async Task<List<string>> GetDependencies(string beamoServiceId, string pr
 		return dependencies;
 	}
 
+	/// <summary>
+	/// Removes the dependency between a microservice and a storage.
+	/// </summary>
+	/// <param name="project">The microservice to have the dependency removed from</param>
+	/// <param name="dependency">The storage to remove as a dependency from the microservice</param>
+	public async Task RemoveProjectDependency(BeamoServiceDefinition project, BeamoServiceDefinition dependency)
+	{
+		if (project.Protocol != BeamoProtocolType.HttpMicroservice ||
+		    dependency.Protocol != BeamoProtocolType.EmbeddedMongoDb)
+		{
+			throw new CliException(
+				$"Currently the only supported dependencies are {nameof(BeamoProtocolType.HttpMicroservice)} depending on {nameof(BeamoProtocolType.EmbeddedMongoDb)}");
+		}
+
+		var relativeProjectPath = _configService.GetRelativePath(project.ProjectDirectory);
+		var projectPath = Path.Combine( relativeProjectPath, $"{project.BeamoId}.csproj");
+		var dependencyPath = Path.Combine( _configService.GetRelativePath(dependency.ProjectDirectory), $"{dependency.BeamoId}.csproj");
+		
+		var command = $"remove {projectPath} reference {dependencyPath}";
+		var (cmd, result) = await CliExtensions.RunWithOutput(_ctx.DotnetPath, command);
+		if (cmd.ExitCode != 0)
+		{
+			throw new CliException($"Failed to remove project dependency, output of \"dotnet {command}\": {result}");
+		}
+		
+		var service = BeamoManifest.HttpMicroserviceLocalProtocols[project.BeamoId];
+		var dockerfilePath = service.RelativeDockerfilePath;
+		dockerfilePath = _configService.GetFullPath(Path.Combine(service.DockerBuildContextPath, dockerfilePath));
+		var dockerfileText = await File.ReadAllTextAsync(dockerfilePath);
+		
+		string search = @$"WORKDIR /subsrc/{dependency.BeamoId}

```

Comment:
can you explain this a bit? We are looking into the dockerfile for this line specifically, and killing it if it exists... 

I think we may want to re-think this a bit. 
I don't love the idea of having the add/remove commands just appending/removing lines arbitrarily from the midst of a dockerfile. Rather, I think we should define a section in the Dockerfile, and wholistically recompute the value of that section based on the known deps everytime. 

So instead of removing this one reference, it would be more like we are re-writing the entire deps section of the Dockerfile. I think that would represent a more stable approach.

Reply:
Right now, when adding a dependency, we are looking for this tag
`# <BEAM-CLI-INSERT-FLAG:COPY> do not delete this line. It is used by the beam CLI to insert custom actions`

And adding the copy reference project part in there when adding it. 

Your suggestion is that we add a new tag just for dependencies, and every time we add/remove, we update it all using that tag to find where to write/delete, right?